.github/ISSUE_TEMPLATE

```diff
@@ -0,0 +1,23 @@
+**Actual Behaviour**
+
+Please state here what is currently happening.
```

Comment:
These comments gives a clear idea of what to write inside these templates. So IMO its better to have it there.